### DISCUSSION TITLE
iOS app laggy when scrolling on 120 Hz device?

### DISCUSSION THREAD

**Discussion by flobernd** [immich-app/immich#12021] [Q&A]
Hi there,

is it just me or does the native iOS app sometimes feels laggy when scrolling on a 120Hz "Pro Motion" phone?

There is no massive stutter that would indicate e.g. background load of images, but rather a very subtile lag. It's like the app uses only 60 Hz instead of 120 Hz.

This can be observed not only in the timeline view, but as well when e.g. horizontally scrolling the "places" etc.

**Comment by radh21301** [comment]
I feel the same way! After getting used to 120Hz, the Immich app feels like it is stuttering -> when compared to the ios photos app.

Wonder if there are ways to test what refresh rate is being used..maybe install an app on the phone that would show some stats. Or maybe some tools can be built into the immich app to test some performance.

**Comment by ne0ark** [comment]
Hello,

I am also experiencing the same issue.

Any resolution?

**Reply by alextran1502** [reply]
@ne0ark it was probably still performing the sync tasks. We are working on a better sync mechanism to avoid causing the ui stuttering when you have a large amount of albums
